### PR TITLE
add php framework DuckPhp

### PR DESCRIPTION
**Pull request by dvaknheo**
first test to add  php framework DuckPhp

### CODE REVIEW

File: frameworks/PHP/duckphp/app/Model/WorldModel.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        
+        return $arr;
+    }
+    public function multiUpdate($query_count)
+    {
+        // For each query, store the result set values in the response array
+        $arr=[];
+        while ($query_count--) {
+            $id = mt_rand(1, 10000);
+            $sql='SELECT randomNumber FROM world WHERE id=?';
```
Comment (joanhey) [inline frameworks/PHP/duckphp/app/Model/WorldModel.php:38]: Need to request id, randomNumber
`Select id, randomNumber FROM world WHERE id=?`

**Review by joanhey — CHANGES_REQUESTED** [review]

**Comment by joanhey** [comment]
Try to clean, and delete all unnecessary files for the benchmark.

**Comment by dvaknheo** [comment]
rip the default config code . and rip some nonessessary files .may look clear .


//
frameworks/PHP/duckphp/app/Model/WorldModel.php 

I port from the base php framework code  
frameworks/PHP/php/updateraw.php  
```php
$statement = $pdo->prepare('SELECT randomNumber FROM World WHERE id=?');
$updateStatement = $pdo->prepare('UPDATE World SET randomNumber=? WHERE id=?');

// For each query, store the result set values in the response array
while ($query_count--) {
    $id = mt_rand(1, 10000);
    $statement->execute([$id]);

    // Store result in array.
    $world = ['id' => $id, 'randomNumber' => $statement->fetchColumn()];
    $world['randomNumber'] = mt_rand(1, 10000);
    $updateStatement->execute(
      [$world['randomNumber'] = mt_rand(1, 10000), $id]
    );

    $arr[] = $world;
}
```

**Comment by joanhey** [comment]
The requirements changed to use id and randomNumber.
Changed in  #6072.

**Comment by dvaknheo** [comment]
stable version.
and ,  can i  have  the beanchmark result. how 
without merge and just show the result to master is enough.
i guess it is better than laravel but less than ubiquity.

**Review by joanhey — APPROVED** [review]

**Comment by nbrady-techempower** [comment]
thanks for your first contribution @dvaknheo and welcome to TFB!

@joanhey as always, thank you for your thorough reviews. It's really appreciated!

**Pull request merged**